src/Uno.UI/UI/Xaml/Automation/AutomationProperties.cs

```diff
@@ -148,6 +149,13 @@ private static void OnAutomationIdChanged(DependencyObject dependencyObject, Dep
 			if (FrameworkElementHelper.IsUiAutomationMappingEnabled && dependencyObject is UIElement uiElement)
 			{
 				uiElement.SetAttribute("xamlautomationid", (string)args.NewValue);
+				uiElement.SetAttribute("aria-label", (string)args.NewValue);
```

Comment:
We should probably calls this anyways if `AutomationId` is set, regardless of IsUiAutomationMappingEnabled being true.

Reply:
@jeromelaban  updated the PR with this suggestion